packages/p2p-v2/src/pages/my-profile/screens/MyProfileCounterparties/MyProfileCounterpartiesTable/MyProfileCounterpartiesTable.scss

```diff
@@ -1,8 +1,9 @@
 .p2p-v2-my-profile-counterparties-table {
-    height: 28rem;
+    max-height: 16rem;
 
     @include mobile {
         height: 100%;
```

Comment:
do you need this height when max-height is already 100%

Reply:
yes, this is to explicitly offset the counterparties infinite list to be 16rem for desktop, otherwise the list would not be scrollable (explicit height needs to be set)